src/openrct2/interface/InteractiveConsole.cpp

```diff
@@ -1488,6 +1489,23 @@ static void ConsoleCommandReplayStart(InteractiveConsole& console, const argumen
         console.WriteFormatLine(logFmt, info.FilePath.c_str(), recordingDate, info.Ticks, info.NumCommands, info.NumChecksums);
         Console::WriteLine(logFmt, info.FilePath.c_str(), recordingDate, info.Ticks, info.NumCommands, info.NumChecksums);
     }
+    else
+    {
+        std::string filePath = OpenRCT2::GetContext()->GetPlatformEnvironment().GetDirectoryPath(
+            OpenRCT2::DirBase::user, OpenRCT2::DirId::replayRecordings);
+        filePath = Path::Combine(filePath, name + ".parkrep");
```

Comment:
This logic is not correct, it only uses the replay directory when its a relative path, when an absolute path is given then it can't combine that. See how it is handled in ReplayManager.cpp